pwnlib/util/proc.py

```diff
@@ -245,6 +245,8 @@ def status(pid):
     try:
         with open('/proc/%d/status' % pid) as fd:
             for line in fd:
+                if -1 == line.find(':'):
```

Comment:
Maybe this looks better?

    if ':' not in line:
        continue

Reply:
Yeah, this would probably be better.  Thanks!